exercitiu3/index.js

```diff
@@ -0,0 +1,6 @@
+const numbers = ['2', '10', '12', '20', '25']
```

Comment:
numerele in general se scriu fara ghilimele.
In exemplu tau acest `numbers` este de fapt o lista de stringuri nu de numere